docs/example_workflows/basic_workflow.yml

```diff
@@ -0,0 +1,14 @@
+name: My workflow
+on:
+  push:
+  pull_request:
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Starknet Foundry
+        uses: foundry-rs/setup-snfoundry@v3
+        
+      - run: snforge --version
```

Comment:
```suggestion
      - run: snforge test
```

Reply:
This will fail because basic workflow doesn't include scarb installation

Reply:
I'd include scarb in both. I think most users will want to use snforge in ci and not sncast.